src/GitHub/Event/GitHubStatus.php

```diff
@@ -21,14 +21,30 @@
  */
 final class GitHubStatus extends AbstractGitHubEvent
 {
+    /**
+     * Patterns for build contexts we're interested in.
+     *
+     * @var string[]
+     */
     private const CONTEXT_PATTERNS = [
+        '#coveralls#',
```

Comment:
Are we interested in coverage report though?

Reply:
We'll only get them if we have failures or errors. Most of our repos don't even have it enabled.